tests/component/genres-input.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/experimental-ct-react';
+import { GenresInputStory } from '@/features/tracks/components/genres-input/genres-input.story';
+
+test('GenresTagInput component test', async ({ mount, page }) => {
+  const component = await mount(<GenresInputStory />);
+
+  await expect(component.getByRole('combobox')).toHaveText('Select genre to add');
+  await expect(component.getByTestId('badge').getByText('Reggae')).not.toBeVisible();
+
+  await component.getByRole('combobox').click();
+  await expect(page.getByText('Reggae')).toBeVisible();
+  await page.getByText('Reggae').click();
+
+  await expect(component.getByTestId('badge').getByText('Reggae')).toBeVisible();
```

Comment:
It's better to move test ids to a constant. As if the id changes you have to manually fix in all places